devicetypes/smartthings/ozom-smart-siren.src/ozom-smart-siren.groovy

```diff
@@ -177,9 +177,7 @@ def strobe() {
 def on() {
 	log.debug "on()"
 
-	def manufacturer = device.getDataValue("manufacturer")
-
-	if (manufacturer == "ClimaxTechnology"){
+	if (isOzomSiren()){
```

Comment:
I'll nit-pick here and at the `} else{`, but there should be a space before the `{`.

Reply:
Actually, there are a few other places that is the case. That and instances of `if(` which should be `if (`.